src/ExpoMessage.php

```diff
@@ -77,17 +71,18 @@ public static function create($body = '')
     /**
      * ExpoMessage constructor.
      *
-     * @param  string  $body
+     * @param string $body
      */
-    public function __construct(string $body = '')
-    {
-        $this->body = $body;
+    public function __construct(
+        protected string $body = '',
+        protected InterruptionLevelEnum $interruptionLevel = InterruptionLevelEnum::Active,
```

Comment:
let's keep the default behaviour 

```suggestion
        protected ?InterruptionLevelEnum $interruptionLevel = null,
```